src/context-selection/org-unit-selector-bar-item/get-offline-levels-to-load.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param {Object} data
+ * @param {Object} data.organisationUnitLevels
+ * @param {Integer} data.organisationUnitLevels.level
+ * @param {Integer} data.organisationUnitLevels.offlineLevels
+ * @param {Object} data.userOrganisationUnits
+ * @param {String} data.userOrganisationUnits.id
+ * @param {Integer} data.userOrganisationUnits.level
+ *
+ * @returns {Array.<{ id: String, offlineLevels: Int }>}
+ */
+export default function getOfflineLevelsToLoad({
+    organisationUnitLevels,
+    userOrganisationUnits,
+}) {
+    return userOrganisationUnits.reduce((acc, userOrgUnit) => {
```

Comment:
I'd replace this with a map and a filter. Reduce is horrible for readability in my opinion.

Reply:
Done